code/procs/gamehelpers.dm

```diff
@@ -115,6 +115,9 @@ var/stink_remedy = list("some deodorant","a shower","a bath","a spraydown with a
 	ENSURE_TYPE(mobuser)
 	if(mobuser?.client?.holder?.ghost_interaction)
 		return TRUE
+	if(!istype(mobuser.loc, /turf)) // if we're not on a turf, we can only interact with stuff inside the same object or in ourselves
+		if(!((source == mobuser.loc) || (source in mobuser.loc) || (source in mobuser)))
+			return FALSE
```

Comment:
how does this work with telekinesis though 😈

Reply:
telekinetic drag works, i pushed a fix that tries to account for the "telekinesis" bioEffect that's explicitly mentioned in code there, would appreciate some more testing from someone familiar with that gene's functions as its not normally obtainable.